tarentula/metadata_fields.py

```diff
@@ -0,0 +1,36 @@
+from json import dumps
+
+from tarentula.datashare_client import DatashareClient
+from tarentula.logger import logger
+
+
+class MetadataFields:
+    def __init__(self,
+                 datashare_url: str = 'http://localhost:8080',
```

Comment:
Since we only do a request to Elasticsearch, I would have avoid instantiating a DatashareClient so you can remove all the parameters that are not used (`datashare_url`, `cookies` and `apikey`).